src/hooks/screen/useChatScreenHook.ts

```diff
@@ -239,7 +273,6 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     goBackFromChatScreen,
     goToChatInfoScreen,
     sendChat,
-    handleUserName,
     updateChatContinuity
   };
 }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [266-278]

The new implementation of the `useEffect` hook is a significant improvement over the previous one. It adds a dependency on `otherListener[`chat_${selectedChannel?.id}`]`, which ensures that `initChatData` is called whenever this value changes. This can enhance performance and maintainability by avoiding unnecessary calls to `initChatData`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->